blackbox/docs/release_notes/1.1.6.txt

```diff
@@ -0,0 +1,63 @@
+.. _version_1.1.6:
+
+=============
+Version 1.1.6
+=============
+
+Released on 2017/06/32.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB 0.57.0 or higher
+   before you upgrade to 1.1.6.
+
+   If you want to perform a :ref:`cluster_upgrade`, your current CrateDB
+   version number must be at least :ref:`version_1.1.1`. Any upgrade from a
+   version prior to this will require a full cluster restart.
+
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Fixes
+=====
+
+ - Fixed an issue that caused an exception to be thrown when applying
+   aggregations on generated columns of a table.
+
+ - Fixed a bug in the memory accounting of the circuit breaker for HTTP
+   results when querying for columns of undefined type.
+
+ - Fixed issue that caused an exception when querying the ``_id`` column using
+   all defined primary keys inside the ``WHERE`` clause over the HTTP API.
+
+ - Fixed wrong results when querying ``IS NULL`` and ``IS NOT NULL`` on an
+   array of objects.
+
+ - Fixed an issue that caused an Exception to be thrown on ``JOIN`` queries
+   with 4 or more tables when an ``ORDER BY`` is also applied.
+
+ - Fixed an issue that resulted in rows could not be queried by primary keys
+   when the order of the primary key columns on insert differs from the order
+   of the primary keys on the table definition.
+   Note: If records are already inserted by using a different primary key
+   column order, they must be re-inserted, otherwise queries will still fail.
+
+ - Improved the resiliency of queries on ``sys.nodes``: If a node disconnects
+   during the execution of a query it will no longer fail.
+
+ - Added proper handling when memory requirements of a ``JOIN`` query exceeds
```

Comment:
This should be reworded, I think. Maybe to:
```
 - Fixed an issue that resulted in rows being unable to be queried by primary
   keys, when the order of the primary key columns on insert differed from
   the order of the primary key columns on the table definition.
   Note: If records are already inserted by using a different primary key
   column order, they must be-reinserted, otherwise queries will still fail
   for these rows.
```